modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,67 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
 ----
-<1> Set the termination field to reencrypt.
-<2> Add the contents of the *_admin-ca_* file.
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
 
-. Use the following command to create the service:
+.. Run the following commnd to add the Elasticsearch CA certificate to the route YAML you created:
```

Comment:
command? ;)

Reply:
And, apparently, need to spell-check, too!